<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.0.3 (Python 3.8.5 on win32)" generated="20210915 09:40:45.583" rpa="false" schemaversion="2">
<suite id="s1" name="TestCommunicationsSpiels" source="E:\Jenkins\Tests\TestCommunicationsSpiels.robot">
<kw name="GlobeLoginKeywords.Open Chrome Goto Login Page" type="SETUP">
<msg timestamp="20210915 09:40:46.223" level="FAIL">No keyword with name 'GlobeLoginKeywords.Open Chrome Goto Login Page' found.</msg>
<status status="FAIL" starttime="20210915 09:40:46.223" endtime="20210915 09:40:46.223"/>
</kw>
<test id="s1-t1" name="Test Communications ${xlbrand} - ${xlmobileno}">
<status status="FAIL" starttime="20210915 09:40:46.225" endtime="20210915 09:40:46.226">Parent suite setup failed:
No keyword with name 'GlobeLoginKeywords.Open Chrome Goto Login Page' found.</status>
</test>
<kw name="OpenCloseBrowserKeywords.CloseTheBrowser" type="TEARDOWN">
<msg timestamp="20210915 09:40:46.233" level="FAIL">No keyword with name 'OpenCloseBrowserKeywords.CloseTheBrowser' found.</msg>
<status status="FAIL" starttime="20210915 09:40:46.232" endtime="20210915 09:40:46.233">No keyword with name 'OpenCloseBrowserKeywords.CloseTheBrowser' found.</status>
</kw>
<doc>Communications Spiels
As a Globe frontliner, I should be able to send spiel(Communications) to the customer.</doc>
<status status="FAIL" starttime="20210915 09:40:45.584" endtime="20210915 09:40:46.233">Suite setup failed:
No keyword with name 'GlobeLoginKeywords.Open Chrome Goto Login Page' found.

Also suite teardown failed:
No keyword with name 'OpenCloseBrowserKeywords.CloseTheBrowser' found.</status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="TestCommunicationsSpiels">TestCommunicationsSpiels</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210915 09:40:45.945" level="ERROR">Error in file 'E:\Jenkins\Tests\TestCommunicationsSpiels.robot' on line 5: Resource file '..\Resources\OpenCloseBrowserKeywords.robot' does not exist.</msg>
<msg timestamp="20210915 09:40:45.947" level="ERROR">Error in file 'E:\Jenkins\Tests\TestCommunicationsSpiels.robot' on line 6: Resource file '..\Resources\GlobeLoginKeywords.robot' does not exist.</msg>
<msg timestamp="20210915 09:40:45.948" level="ERROR">Error in file 'E:\Jenkins\Tests\TestCommunicationsSpiels.robot' on line 7: Resource file '..\Resources\CustomKeywords.robot' does not exist.</msg>
<msg timestamp="20210915 09:40:46.202" level="ERROR">Error in file 'E:\Jenkins\Tests\TestCommunicationsSpiels.robot' on line 11: Test library '..\libraries\Spiels.py' does not exist.</msg>
<msg timestamp="20210915 09:40:46.214" level="ERROR">Error in file 'E:\Jenkins\Tests\TestCommunicationsSpiels.robot' on line 12: Importing library 'DataDriver' failed: ModuleNotFoundError: No module named 'DataDriver'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\ALEX\anaconda3\Scripts\robot.exe
  c:\users\alex\anaconda3\python38.zip
  c:\users\alex\anaconda3\DLLs
  c:\users\alex\anaconda3\lib
  c:\users\alex\anaconda3
  c:\users\alex\anaconda3\lib\site-packages
  c:\users\alex\anaconda3\lib\site-packages\win32
  c:\users\alex\anaconda3\lib\site-packages\win32\lib
  c:\users\alex\anaconda3\lib\site-packages\Pythonwin</msg>
</errors>
</robot>
